docs/getting-started/getting-started-with-notebooks.rst

```diff
@@ -0,0 +1,164 @@
+    :width: 60 %
+
+Next, we will set up a notebook on Google Colab so that it's also stored in one of your GitHub repositories. If you already have a notebook in Google Colab that's saved to a GitHub repository, you may proceed to :ref:`create the .dockstore.yml file for your repository<create-dockstore-yml>`.
+
+We will cover two scenarios:
+
+1. :ref:`You don't have a notebook in GitHub<no-notebook-in-github>`. You may or may not have a notebook in Google Colab.
+2. :ref:`You have a notebook in GitHub, but it's not in Google Colab<no-notebook-in-google-colab>`
+
+.. _no-notebook-in-github:
+
+You don't have a notebook in GitHub
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you don't have a notebook in GitHub *nor* Google Colab, navigate to Google Colab and create a new notebook by clicking on File > New notebook. Edit your new notebook if you like.
+
+If you already have a notebook in Google Colab, proceed to the next step.
+
+Click on File > Save a copy in GitHub to save your notebook to a GitHub repository. You may want to create a new repository for your notebook if you don't already have a repository that you want to save the notebook to.
+
+.. figure:: /assets/images/docs/google-colab-save-on-github.png
+    :width: 35 %
+
+Configure where you want to save the notebook to on GitHub then click OK. Google Colab will push a commit to your repository with the notebook.
+
+.. figure:: /assets/images/docs/google-colab-save-on-github-dialog.png
+    :width: 60 %
+
+.. _no-notebook-in-google-colab:
+
+You have a notebook in GitHub, but it's not in Google Colab
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Click on File > Open notebook then select the GitHub tab in the dialog box. Select the notebook that you want to import from GitHub. This will open the notebook in Google Colab.
+
+.. figure:: /assets/images/docs/google-colab-open-notebook-github.png
+    :width: 60 %
+
+.. _create-dockstore-yml:
+
+Create Your .dockstore.yml File
+-------------------------------
+
+The first step is to create a file named ``.dockstore.yml`` which acts as a configuration file used to describe your notebook. We will cover an example .dockstore.yml written for a simple hello world Jupyter notebook.
+You can also view template .dockstore.yml files :doc:`here </assets/templates/notebooks/notebooks>` that describe the possible fields and values you can use to configure your notebook.
+
+.. include:: /assets/templates/notebooks/example-1-name.yml
+  :code:
+
+Line 1 specifies the ``.dockstore.yml`` version. You should use the latest version, which is version 1.2.
+
+Next is a required key named ``notebooks`` where your notebooks will be configured.
+
+Within this, you need to specify the ``format`` and ``language`` of the notebook. Currently, we only support ``JUPYTER`` notebooks. For a list of all the languages supported, view the :doc:`notebook template files </assets/templates/notebooks/notebooks>`.
```

Comment:
See above, `format` and `language` are optional and default to `Jupyter` and `Python`.